src/datatable.js

```diff
@@ -195,7 +191,7 @@ export class DataTable {
       labels.push({
         nested: cleanedColumn.indexOf('.') !== -1,
         column: cleanedColumn,
-        label : ucfirst(clean(aliased[1] || aliased[0]))
+        label : clean(aliased[1] || aliased[0])
```

Comment:
We'll need something here to make this configurable and enabled by default. Until that time I suggest adding ucfirst to form or translating the keys twice. I don't like the concept of having to alias all my keys.